windows-containers/productization.md

```diff
@@ -0,0 +1,180 @@
+---
+title: windows-containers-productization
+authors:
+  - "@ravisantoshgudimetla"
+  - "@aravindhp"
+reviewers:
+  - "@crawford"
+  - "@sdodson"
+approvers:
+  - "@crawford"
+  - "@sdodson"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: implementable
+---
+
+# Windows Containers Productization
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The intent of this enhancement is to allow a cluster administrator to add a
+Windows worker node with a prescribed configuration to an OpenShift cluster and
+enable scheduling of Windows workloads.
+
+## Motivation
+
+The main motivation behind this enhancement is to satisfy customer
+requirement of being able to run Windows workloads on OpenShift clusters.
+
+### Goals
+
+As part of this enhancement we plan to do the following:
+* Prepare an already provisioned Windows worker node to join the cluster
+* Perform all the required steps within the node for it to be a Kubernetes
+worker
+* Upgrade of all OpenShift related components in the node
+
+### Non-Goals
+
+As part of this enhancement we do not plan to support:
+* Windows node provisioning or de-provisioning
+* Installing the container runtime in the Windows node
+* Windows operating system upgrades
+* Node management like reboots and node draining
+* OpenShift Builds and DeploymentConfigs
+
+## Proposal
+
+The two main pieces that will allow us to achieve our goals is a Windows Scale
+Up (WSU) Ansible playbook and a Windows Machine Config Bootstrapper (WMCB)
+binary. The Ansible playbook will collect information from the cluster and
```

Comment:
Shouldn't we explain why we're distributing a binary?

Reply:
Are you implying that I should mention the issue with building Windows containers? Or are you referring to something else?

Reply:
Yes, I think we should explicitly tell that we're not building windows containers anywhere

Reply:
I have addressed this.